### PR TITLE
Added English editorial for LightOj 1428 (Melody Comparison)

### PR DESCRIPTION
**Pull request by zarif98sjs**

### CODE REVIEW

File: 1428/en.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
+
+<h1 align="center"> <b>LightOj 1428 (Melody Comparison) </b> </h1>
```
Comment (faiyaz26) [inline 1428/en.md:2]: Add the title like this: https://github.com/lightoj-dev/problem-tutorials/pull/11/files#diff-49d89fcefb3733849902cae167206bfef3d2d2fa1569a2abcc5b303e53f828b9R1

File: 1428/en.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+<hr/>
+
+# **Tags**
```
Comment (faiyaz26) [inline 1428/en.md:5]: use header2 on markdown, its double hash

File: 1428/en.md [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+
+**`string`** , **`suffix array`** , **`binary search`** , **`kmp`**
```
Comment (faiyaz26) [inline 1428/en.md:7]: dont need to make it bold

File: 1428/en.md [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+
```
Comment (faiyaz26) [inline 1428/en.md:8]: Explain the problem a bit, what the problem is asking

File: 1428/en.md [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+&nbsp;
+# **Idea**
```
Comment (faiyaz26) [inline 1428/en.md:10]: Only for the problem title use header1 markdown, for other header 2 and so on

**Review by faiyaz26 — CHANGES_REQUESTED** [review]

**Comment by zarif98sjs** [comment]
Change requests are resolved now . Please check if everything is okay now .

**Pull request merged**